Makefile

```diff
@@ -17,12 +17,22 @@ endif
 .PHONY: all install copyfiles version plugins dependencies sshcommand pluginhook docker aufs stack count dokku-installer vagrant-acl-add vagrant-dokku
 
 include tests.mk
+include deb.mk
 
 all:
 	# Type "make install" to install.
 
 install: dependencies stack copyfiles plugin-dependencies plugins version
 
+release:
+	$(MAKE) deb-all
+	package_cloud push dokku/dokku/ubuntu/trusty buildstep*.deb
+	package_cloud push dokku/dokku/ubuntu/trusty sshcommand*.deb
+	package_cloud push dokku/dokku/ubuntu/trusty pluginhook*.deb
+	package_cloud push dokku/dokku/ubuntu/trusty rubygem*.deb
+	package_cloud push dokku/dokku/ubuntu/trusty dokku*.deb
+	packer build contrib/packer.json
+
```

Comment:
This may be pedantic but I like the idea of 1 target for `package_cloud` and 1 for `packer`. Then `release` just deps those 2. In case one fails and the other doesn't and you want to just re-run one. Just a thought.